archaius2-core/src/main/java/com/netflix/archaius/DefaultDecoder.java

```diff
@@ -63,7 +63,7 @@
             else if (v.equalsIgnoreCase("false") || v.equalsIgnoreCase("no") || v.equalsIgnoreCase("off")) {
                 return Boolean.FALSE;
             }
-            throw new ParseException("Error parsing value '" + v, new Exception("Expected one of [true, yes, on, false, no, off]"));
+            throw new ParseException("Error parsing value '" + v + "'", new Exception("Expected one of [true, yes, on, false, no, off]"));
```

Comment:
Out of curiosity, why does it create a new exception with the expected values just to wrap it? Couldn't the expectation just be appended to the message for the ParseException?